cctools/as/i386.c

```diff
@@ -5408,6 +5409,8 @@ int md_long_jump_size = 5;
 /* Size of relocation record.  */
 const int md_reloc_size = 8;
 
+void md_create_short_jump (char *ptr, addressT from_addr, addressT to_addr, fragS *, symbolS *to_symbol);
```

Comment:
So, what I'd do here would be to move the newly-added prototypes to `i386.h`, since they're extern, and then update the function definitions here so that they're no longer old-style.